dipy/reconst/dti.py

```diff
@@ -858,6 +877,16 @@ def quadratic_form(self):
         # np.einsum('...ij,...j,...kj->...ik', evecs, evals, evecs)
         return vec_val_vect(self.evecs, self.evals)
 
+    @property
+    def S0(self):
```

Comment:
Why do we need to have a self.S0 and a self.S0_hat?
Can we just have S0 (i.e. define S0_hat as S0 in line 834 and delete this property)?